src/app/(proper_react)/redesign/(public)/WalkthroughImages.tsx

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ExtendedReactLocalization } from "../../../hooks/l10n";
+import Image from "next/image";
+import ScanningForExposuresImage from "./value-prop-images/scanning-for-exposures.svg";
+import LeakedPasswordExampleImage from "./value-prop-images/leaked-password-example.svg";
+import ScanningForExposuresImageDe from "./value-prop-images/de/scanning-for-exposures-de.svg";
+import LeakedPasswordExampleImageDe from "./value-prop-images/de/leaked-password-example-de.svg";
+import ScanningForExposuresImageFr from "./value-prop-images/fr/scanning-for-exposures-fr.svg";
+import LeakedPasswordExampleImageFr from "./value-prop-images/fr/leaked-password-example-fr.svg";
+
+const IllustrationWrapper = ({
+  image,
+  l10n,
+  testId,
+  countryCode,
+}: {
+  image: string;
+  l10n: ExtendedReactLocalization;
+  testId: string;
+  countryCode: string;
+}) => (
+  <Image
+    src={image}
+    alt={l10n.getString(
+      "landing-all-value-prop-scanning-for-exposures-illustration-alt",
+    )}
+    data-testid={testId}
+    country-code={countryCode}
```

Comment:
`country-code` isn't a valid HTML attribute, I think?

What _is_ a valid attribute is [`lang`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang), so presumably that would be more appropriate here?

Reply:
Or `data-country-code=` if we want to use custom attributes.